optimizer/plan/cost.go

```diff
@@ -29,49 +29,46 @@ const (
 )
 
 // CostEstimator estimates the cost of a plan.
-type costEstimator struct {
-}
 
 // Enter implements Visitor Enter interface.
-func (c *costEstimator) Enter(p Plan) (Plan, bool) {
-	return p, false
-}
-
-// Leave implements Visitor Leave interface.
-func (c *costEstimator) Leave(p Plan) (Plan, bool) {
+func estimate(p Plan) {
+	var child Plan
```

Comment:
well, here is a hack. cause all the operator needed to be processed only have a single child. cost.go is an incomplete code, will be continued to optimize.